install/troubleshooting_utility_storage_node_installation.md

```diff
@@ -65,22 +64,22 @@ ID CLASS WEIGHT   TYPE NAME         STATUS REWEIGHT PRI-AFF
 
 1. Put in safeguard.
 
-   * Edit `/srv/cray/scripts/metal/lib.sh`
-   * Comment out the below lines
+   - Edit `/srv/cray/scripts/metal/lib.sh`
+   - Comment out the below lines
 
    ```bash
```

Comment:
Or we could take out the line number prefixes and leave it as bash

```suggestion
   ```text
```

Reply:
Yup. You could instead change the line above it to say "Comment out lines 22-24", remove the line numbers from the fenced code block, and then leave it as bash.

Reply:
This is wrong.. I can't find this script on 1.2 NCNs:

```bash
ncn-s001:~ # vim /srv/cray/scripts/metal/
generate_haproxy_cfg.sh      generate_keepalived_conf.sh  install.sh                   lib-1.4.sh                   lib-1.5.sh                   metal-lib.sh                 net-init.sh                  ntp-upgrade-config.sh        set-ntp-config.sh            update_apparmor.sh
```

Reply:
there is no lib-1.4.sh anymore.  there shouldn't be a lib-1.5.sh anymore either.  just lib.sh

Reply:
def get rid of the line numbers.  it should be a paste-able bash script

Reply:
@cdelatte-hpe in 1.3 these lines do not exist in the `lib.sh` file on storage nodes.